.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
```

Comment:
```suggestion
# Adapted from https://github.com/rapidsai/cugraph/blob/main/.github/CODEOWNERS
```

should this be `cugraph` instead of `cudf`?

Reply:
Cugraph doesn't have a code owners file. So it's actually adapted from the code owners in `cudf`.